client/js/components/ParkMap.jsx

```diff
@@ -1,18 +1,31 @@
+import $ from 'jquery';
+import React from 'react';
+import { GeoJson, Map, Marker, Popup, TileLayer } from 'react-leaflet';
 
-class Park extends React.Component {
+export default class ParkMap extends React.Component {
     constructor(props) {
         super(props);
-        var pID = this.props.parkId;
     }
 
     render () {
+        var center = this.props.center;
+        var p = this.props
```

Comment:
If the code using it weren't on the next three or four lines I might agree. I'm refactoring this a little anyway and the problem has disappeared.